accounts/erd1qqqqqqqqqqqqqpgqgdf6vk43c2jxk4a6nw2adv8vmnpqagegtxfqmf8et2.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Middle Staking : Stake contract",
+  "description": "Stake $MID and other tokens to earn rewards",
+  "social": {
+    "website": "https://www.middlestaking.fr/",
+    "twitter": "MiddleStaking",
+    "telegram": "https://t.me/middlestaking"
+  },
+  "tags": ["middlestaking", "staking"],
```

Comment:
please add also the contract tag 🙏

Reply:
["middlestaking", "staking", "nft"] -> ["middlestaking", "staking", "nft", "contract"]